src/Header/HttpListener.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_HTTPLISTENER_H_
+#define GROUP4_HEADER_HTTPLISTENER_H_
+
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <netinet/in.h>
+#include <memory.h>
+
+#include <thread>
```

Comment:
Please order the include.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done